server/service/hosts.go

```diff
@@ -393,7 +397,11 @@ type hostByIdentifierRequest struct {
 
 func hostByIdentifierEndpoint(ctx context.Context, request interface{}, svc fleet.Service) (interface{}, error) {
 	req := request.(*hostByIdentifierRequest)
-	host, err := svc.HostByIdentifier(ctx, req.Identifier, false)
+	opts := fleet.HostDetailOptions{
+		IncludeCVEScores: false,
+		IncludePolicies:  true,
```

Comment:
`IncludePolicies` is `true` here to preserve existing behavior of the hosts endpoint.

Reply:
Just to make sure I got it right, when fetching a single host we want to include Policies regardless whether they are premium or not, but when listing multiple hosts we only want to show policies if they are premium?

Reply:
This is existing behavior, and at least for now, yes. 

We briefly discussed it on Slack and the idea of this task is to change only the "device" endpoints. cc: @chiiph should I create a follow-up task for hosts endpoint, or they are good?

Reply:
I would suggest leaving a comment here explaining why it's true here. Someone could think it's a bug.